README.md

```diff
@@ -136,21 +140,68 @@ See the [CONTRIBUTING](https://github.com/OpenTermsArchive/contrib-declarations/
 
 ### Installing
 
-This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v14.x](https://nodejs.org/en/download/) to run it.
-
-Clone the repository and install dependencies:
+- This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v14.x](https://nodejs.org/en/download/) to run it.
+- When refering to the root folder, it means the folder where you will be `git pull`ing everything.
```

Comment:
This is ambiguous: “root” has a specific meaning on the filesystem. I would prefer that we avoid naming this folder. If we do need it though, I believe that “work folder” or “base folder” would be more appropriate 🙂